tests/fsharp/typecheck/sigs/temp

```diff
@@ -0,0 +1,116 @@
+
```

Comment:
Are those here on purpose, in the temp directory?

Reply:
removed